### ISSUE TITLE
`ter-indent` does not work for Typescript `.tsx` files

### ISSUE THREAD

**Issue by mattnedrich** [buzinas/tslint-eslint-rules#209]
The `ter-indent` indentation rules do not seem to work for my the `jsx` code in my `.tsx` files. Has anyone else run into this issue?

**Comment by jmlopez-rod** [comment]
Can you give a simple example where this is failing? The code is trying to emulate the eslint rule and I think they cover support for jsx code. At the moment, the rule is a bit behind and it is in need of being updated.

**Comment by mattnedrich** [comment]
It seems to be failing for something simple like this:

```jsx
class MyApp extends React.Component<any, any> {
  const normalIndentVar = 5;
    const extraIntentVar = 6;
  public render() {
    return (
      <div>
        <NormalIndentComponent />
          <ExtraIndentComponent />
      </div>
    )
  }
}

export default MyApp;
```

Assuming `ter-indent` is set for 2 space indentation, it will flag the `const extraIndentVar` as being indented too much, but not the `<ExtraIndentComponent />`.

**Comment by L0stSoul** [comment]
Have the same issue :(

**Comment by ycjcl868** [comment]
Same issue

**Comment by lokoArt** [comment]
Same problem... Any updates?
